src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -97,8 +104,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
         }
 
-        data.comment = feedbackResponseComment;
-
+        // createdAt, lastEditedAt, commentGiverName and lastEditorName are required to generate editedCommentDetails
+        data.comment = updatedComment;
+        data.question = logic.getFeedbackQuestion(response.feedbackQuestionId);
+        data.showCommentToString = StringHelper.toString(updatedComment.showCommentTo, ",");
+        data.showGiverNameToString = StringHelper.toString(updatedComment.showGiverNameTo, ",");
+        data.instructorEmailNameTable = instructorEmailNameTable;
```

Comment:
@whipermr5 We need `getEditedAtText` function of `FeedbackResponseCommentRow`.As the function is `private` we need to create `FeedbackResponseCommentRow`. Changing that function to `public static` will also demand variables used in the function to be `static`, which I think should be avoided.
Another approach would be to change `getEditedAtText` function to `public static` and pass `instructorEmailNameTable`, `sessionTimeZone` and `commentGiverName` as parameters.
Your input?